kaitmore/snippets
# add basic app skeleton

## Changes
- generated app skeleton with electron-forge
- added a listener for hot key press
- added data store and store schema

## To test
- `npm start`
- highlight some text and press `ctrl+x`
- open `~/Library/Application\ Support/snippet/config.json` and see that the selection was persisted
